tests/lib/rules/semi-spacing.js

```diff
@@ -136,7 +159,31 @@ ruleTester.run("semi-spacing", rule, {
             code: "var a = 'b'; c = 'd';",
             output: "var a = 'b';c = 'd';",
             options: [{ before: false, after: false }],
-            errors: [{ messageId: "unexpectedWhitespaceAfter", type: "VariableDeclaration", line: 1, column: 12 }]
+            errors: [
+                {
+                    messageId: "unexpectedWhitespaceAfter",
+                    type: "VariableDeclaration",
+                    line: 1,
+                    column: 13,
+                    endColumn: 13,
+                    endLine: 1
+                }
+            ]
+        },
+        {
+            code: "var a = 'b';   c = 'd';",
+            output: "var a = 'b';c = 'd';",
+            options: [{ before: false, after: false }],
+            errors: [
+                {
+                    messageId: "unexpectedWhitespaceAfter",
+                    type: "VariableDeclaration",
+                    line: 1,
+                    column: 13,
+                    endColumn: 15,
+                    endLine: 1
+                }
+            ]
```

Comment:
Can we flip `endColumn` and `endLine` lines, to have the same order as with `line`, `column`